src/main/java/org/onebusaway/alexa/AuthedSpeechlet.java

```diff
@@ -73,6 +74,7 @@ public SpeechletResponse onIntent(final IntentRequest request,
                             userData.getCity(), userData.getRegionName()));
             return SpeechletResponse.newTellResponse(out);
         } else if (SET_STOP_NUMBER.equals(intent.getName())) {
+            populateAttributes(session);
```

Comment:
I think AuthedSpeechlet should `populateAttributes` always, rather than just in this branch. That'll make it easier for us to reason about session attributes.